chia/wallet/puzzles/did_innerpuz.clvm

```diff
@@ -118,10 +118,17 @@
   ;MAIN
   (if mode
     ; mode 1 - run INNER_PUZZLE
-    (a INNER_PUZZLE my_amount_or_inner_solution)
+    (if (= mode 2)
+       ; If mode is 2, it means this is a transfer spend.
+       (if (= (sha256tree1 recovery_list_reveal) RECOVERY_DID_LIST_HASH)
```

Comment:
Why do we want to force this?
If the user wants to reveal it when they transfer the wallet then they can.

There are situations where they might not want to reveal the list.

Reply:
I want to add a check in case of user reveals wrong recovery list. It was (if recovery_list_reveal), but seems it require use set 0 for recovery_list_reveal